monitoring/aws/install-vars.json

```diff
@@ -4,7 +4,8 @@
         "arane",
         "efried",
         "mworthin",
-        "sumehta"
+        "sumehta",
+        "mold"
```

Comment:
Please preserve alpha order.

Not because it's really important. Just to exercise doing another rev. (Force push over the same commit, don't add another one.)